### PR TITLE
BA-1617-001: Renaming Operations to Student Services

### PR DESCRIPTION
**Pull request by iangmhill**
**BA-1617-001:** An amendment renaming the CORe Representative to Operations to Representative for Student Services.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ Not yet amended.
 ## Student Government By-Laws
 Originally passed at a meeting of the Student Government on May 1st, 2015.
 ### Passed Amendments
+- 2016-10-05 BA-1617-001: Replaced "Representative to Operations" with "Representative for Student Services"
```
Comment (sammyers) [inline README.md]: Shouldn't this be Campus Services?

Comment (iangmhill) [inline README.md]: You're absolutely right, we did. Fixing...

File: student_government_bylaws.md [hunk 2]
```diff
@@ -92,7 +92,7 @@ Five of the ten representatives are Specialized Representatives who each meet
 directly with senior administrators of the College on a regular basis. These
 representatives include:
 
-1. a Representative to Operations
+1. a Representative for Student Services
```
Comment (sammyers) [inline student_government_bylaws.md]: Also here

**Comment by iangmhill** [comment]
Changes fixed

**Pull request merged**